Implementacija/savetovaliste/app/Views/postavi_pitanje.php

```diff
@@ -64,186 +69,184 @@
                         <div class="input letters_dark_blue">
                             <h4 class="letters_dark_blue">Izaberite kategoriju pitanja</h4>
 
-                            <form action="">
+                            <form name="postaviPitanje"  action="<?= site_url("$controller/postaviPitanje") ?>">
```

Comment:
Valjalo bi da method bude post da se ne prenose argumenti gore url.
```suggestion
                            <form name="postaviPitanje"  action="<?= site_url("$controller/postaviPitanje") ?>" method="post">
```

Reply:
Vazi